modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -261,6 +262,26 @@ private static void addByteAsHex(StringBuilder sb, byte b) {
         sb.append(Integer.toHexString(MASK & b >>> 4)).append(Integer.toHexString(MASK & b));
     }
 
+    /**
+     * Returns a hex string representation of the given long value.
+     *
+     * @param val Value to convert to string.
+     * @return Hex string.
+     */
+    public static String hexLong(long val) {
+        return new IgniteStringBuilder(16).appendHex(val).toString();
+    }
+
+    /**
+     * Rtturns a hex string representation of the given integer value.
+     *
+     * @param val Value to convert to string.
+     * @return Hex string.
+     */
+    public static String hexInt(int val) {
+        return new IgniteStringBuilder(8).appendHex(val).toString();
```

Comment:
`Integer.toHexString(int)`?

Reply:
No, unfortunately

Reply:
These methods add 0 in the beginning so that the resulting string has fixed width